assets/dev/js/editor/editor-base.js

```diff
@@ -1210,6 +1210,12 @@ export default class EditorBase extends Marionette.Application {
 	onEditModeSwitched() {
 		const activeMode = this.channels.dataEditMode.request( 'activeMode' );
 
+		dispatchEvent( new CustomEvent( 'elementor/v1/edit-mode/change', {
```

Comment:
Talk with @rami-elementor

Reply:
According to the current convention it should be `elementor/edit-mode/change`.

Reply:
Please let's not change the conventions again. We have many inconsistencies, we don't need another one.